UKHO.MaritimeSafetyInformation.Web/UKHO.MaritimeSafetyInformation.Common/Helpers/NMHelper.cs

```diff
@@ -97,5 +97,19 @@ public static void ValidateParametersForDownloadSingleFile(List<KeyValuePair<str
                 }
             }
         }
+
+        public static void ValidateParametersForDailyFile(List<KeyValuePair<string, string>> parameters, string correlationId, ILogger logger)
```

Comment:
Please remove this unused method as existing common method 'ValidateParametersForDownloadSingleFile' is used for validation.

Reply:
removed unused function